crates/matrix-sdk-ui/src/timeline/event_item/content/message.rs

```diff
@@ -237,6 +237,13 @@ impl InReplyToDetails {
 
         InReplyToDetails { event_id, event: TimelineDetails::from_initial_value(event) }
     }
+
+    pub fn new_from_event(
```

Comment:
Please add missing documentation.

Also, what about this naming:

```suggestion
    pub fn new_with_timeline_details(
```